src/test/codegen/align-struct.rs

```diff
@@ -10,6 +10,7 @@
 
 // compile-flags: -C no-prepopulate-passes
 // ignore-tidy-linelength
+// min-llvm-version 7.0
```

Comment:
Can you explain why a min-llvm-version is introduced here? Is this regressing support for older versions?

Reply:
Oh sure yeah! This isn't actually regressing compiled-object-code support but it's "regressing" the IR-level FileCheck support. Or in other words the syntax of the LLVM IR changed (alignment of pointers moved around). Instead of figuring out a regex that passed both LLVM 6 and LLVM 7 I figured it'd be easiest to just only run these tests on LLVM 7